addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -92,4 +92,80 @@ tour.register('edit_megamenu', {
         run: function () {}, // it's a check
     },
 ]);
+tour.register('edit_megamenu_big_icons_subtitles', {
+    test: true,
+    url: '/?enable_editor=1',
+}, [
+    // Add a megamenu item to the top menu.
+    {
+        content: "Click on a menu item",
+        trigger: '#top_menu .nav-item a',
+    },
+    {
+        content: "Click on 'Link' to open Link Dialog",
+        trigger: '.o_edit_menu_popover a.js_edit_menu',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+        extra_trigger: ".o_web_editor_dialog:visible",
+        trigger: '.modal-body a.js_add_menu[data-type="mega"]',
+    },
+    {
+        content: "Write a label for the new menu item",
+        extra_trigger: ".o_link_dialog",
+        trigger: '.o_link_dialog #o_link_dialog_label_input',
+        run: 'text Megaaaaa!',
+    },
+    {
+        content: "Save the new menu item",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
+        run: 'click',
+    },
+    {
+        content: "Save the changes to the menu",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
+        run: 'click',
+    },
+    // Edit a menu item
+    wTourUtils.clickOnEdit(),
```

Comment:
This is probably a source of race condition, since you click on edit right after having click on "save", but you actually don't await the modal save page reload, meaning you will have 2 concurrents rpc: it could click on "edit" before the rpc was back/processed
There should be an "check step" which check for the new menu in the menu (meaning the page has fully reload)